controllers/managedfusionoffering_controller.go

```diff
@@ -56,6 +56,7 @@ type ManagedFusionOfferingReconciler struct {
 //+kubebuilder:rbac:groups=misf.ibm.com,resources={managedfusionofferings,managedfusionofferings/finalizers},verbs=get;list;watch;create;update;patch;delete
 //+kubebuilder:rbac:groups=misf.ibm.com,resources=managedfusionofferings/status,verbs=get;update;patch
 //+kubebuilder:rbac:groups="operators.coreos.com",resources={subscriptions,operatorgroups,catalogsources},verbs=get;list;watch;create;update;patch;delete
+//+kubebuilder:rbac:groups="network.openshift.io",resources=egressnetworkpolicies,verbs=create;get;list;watch;update
```

Comment:
Isn't this change needs an ingress policy RBAC and not an egress policy RBAC?
Also if we want to have RBAC for an offering specific resource we should move this line to the datafoundation package

Reply:
Yes, updated accordingly